scripts/customize_conda.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+# This customization-script can be used to customize the conda environment.
+# It expects the following metadata:
+#   conda-component: (Required) Must be either ANACONDA or MINICONDA3.
+#   conda-env-config-uri: (Optional) Must be a gsutil path to the yaml config
+#   file.
+#   conda-packages: (Optional) A list of conda packages with versions to be
+#   installed in the base environment. Must be of the format
+#   <pkg1>:<version1>_<pkg2>:<version2>...
+#   Example: conda-packages=pytorch:1.4.0_visions:0.7.1
+#   pip-packages: (Optional) A list of pip packages with versions to be
+#   installed in the base environment. Must be of the format
+#   <pkg1>:<version1>_<pkg2>:<version2>...
+#   Example: tokenizers:0.10.1_datasets:1.5.0
```

Comment:
You want to give command-line full examples for both config-uri case and package list case.

Reply:
Done.